localstack/config.py

```diff
@@ -933,6 +933,10 @@ def populate_edge_configuration(
 LAMBDA_LIMITS_MAX_FUNCTION_ENVVAR_SIZE_BYTES = int(
     os.environ.get("LAMBDA_LIMITS_MAX_FUNCTION_ENVVAR_SIZE_BYTES", 4 * 1024)
 )
+# SEMI-PUBLIC: not actively communicated
+LAMBDA_LIMITS_MAX_FUNCTION_PAYLOAD_SIZE_BYTES = int(
+    os.environ.get("LAMBDA_LIMITS_MAX_FUNCTION_PAYLOAD_SIZE_BYTES", 6 * 1024 * 1024 + 100)
```

Comment:
FYI: There are even more different quotes: https://docs.aws.amazon.com/lambda/latest/dg/gettingstarted-limits.html
just in case we run into these in the future ...

> 6 MB each for request and response (synchronous)
>
> 20 MB for each [streamed response](https://docs.aws.amazon.com/lambda/latest/dg/configuration-response-streaming.html) (Synchronous. The payload size for streamed responses can be increased from default values. Contact AWS Support to inquire further.)
> 
> 256 KB (asynchronous)